packages/hint-compat-api/src/helpers/compat-api.ts

```diff
@@ -47,25 +51,25 @@ export class CompatApi {
         return namespaceFeatures;
     }
 
-    private filterFeatureByBrowsers(featureValue: CompatStatement & MDNTreeFilteredByBrowsers, browsers: BrowserSupportCollection): CompatStatement | null {
+    private filterFeatureByBrowsers(featureValue: CompatStatement & MDNTreeFilteredByBrowsers): CompatStatement | null {
```

Comment:
are you filtering something here? or you are getting  if have feature?

Reply:
👍